docs/migration.md

```diff
@@ -371,15 +388,36 @@ processors:
     source_category: example category
     ## Installed Collector substitute for `Source Host`.
     source_host: example host
+  transform/logs source:
+    log_statements:
+      ## By default every data has timestamp (usually set to receipt time)
+      ## and therefore Sumo Logic backend do not try to parse it from log body.
+      ## Using this processor works like `Enable Timestamp Parsing`,
+      ## where `Time Zone` is going to be taken from `extensions` section.
+      ## There is no possibility to configure several time zones in one exporter.
+      ## It behaves like `Timestamp Format` would be set to `Automatically detect the format`
+      ## in terms of Installed Collector configuration.
+      - context: log
+        statements:
+          - set(time_unix_nano, 0)
+      ## Adds custom fields:
+      ## - cloud.availability_zone=zone-1
+      ## - k8s.cluster.name=my-cluster
+      - context: resource
+        statements:
+        - set(attributes["cloud.availability_zone"], "zone-1")
+        - set(attributes["k8s.cluster.name"], "my-cluster")
+  ## Remove logs with timestamp before Sat Dec 31 2022 23:00:00 GMT+0000
+  ## This configuration covers `Collection should begin` functionality.
+  ## Please ensure that timestamps are correctly set (eg. use operators in filelog receiver)
+  filter/remove older:
+    logs:
+      log_record:
+      ## - 1672527600000000000 ns is equal to Dec 31 2022 23:00:00 GMT+0000
+      ## - do not remove logs which do not have correct timestamp
+      - 'time_unix_nano < 1672527600000000000 and time_unix_nano > 0'
 exporters:
   sumologic:
-    ## clear_logs_timestamp is by default set to True.
-    ## If it's set to true, it works like `Enable Timestamp Parsing`,
-    ## and `Time Zone` is going to be taken from `extensions` section.
-    ## There is no possibility to configure several time zones in one exporter.
-    ## clear_logs_timestamp sets to true also behaves like
-    ## `Timestamp Format` would be set to `Automatically detect the format`
-    ## in terms of Installed Collector configuration.
     clear_logs_timestamp: true
```

Comment:
Shouldn't we delete this config value as well? In fact, I think we should delete the whole exporter section.

Reply:
We cannot delete the whole exporter section, as

```
Error: invalid configuration: no exporter configuration specified in config
2023/03/09 08:24:24 collector server run finished with error: invalid configuration: no exporter configuration specified in config
```

Reply:
My bad, from context I thought these were just snippets and not full configurations.

Reply:
Maybe this should be changed. For now it's building configuration step by step